.ci/publish-nightly.yml

```diff
@@ -21,6 +21,7 @@ extends:
     vscePackageArgs: --no-dependencies
     cgIgnoreDirectories: 'testdata,demos,.vscode-test'
     l10nShouldProcess: false
+    ghCreateTag: false
```

Comment:
does this help fix unit tests?

Reply:
oh, intentional thing. The pipeline default changed and started adding tags which is really messy